src/abstractdataframe/abstractdataframe.jl

```diff
@@ -596,7 +596,9 @@ completecases(df::AbstractDataFrame, cols::Union{AbstractVector, Regex, Not}) =
 
 """
     dropmissing(df::AbstractDataFrame; disallowmissing::Bool=true)
-    dropmissing(df::AbstractDataFrame, cols; disallowmissing::Bool=true)
+    dropmissing(df::AbstractDataFrame, ::Colon; disallowmissing::Bool=true)
```

Comment:
Why not merge this with the one below? Or even merge the three last signatures?

Also need to break before `;`.

Reply:
fixed